src/main/java/jnr/ffi/Struct.java

```diff
@@ -239,6 +245,34 @@ public double doubleValue() {
         }
     }
 
+    public static final class Alignment extends Number {
```

Comment:
This is just a tiny type and it was based on the Offset type. I just followed the same design. But if you prefer I can replace it by an 'int' or 'long' without problems.

https://github.com/jnr/jnr-ffi/blob/master/src/main/java/jnr/ffi/Struct.java#L224